jupyter_client/tests/signalkernel.py

```diff
@@ -56,13 +56,13 @@ def kernel_info_request(self, *args, **kwargs):
         
         triggers slow-response code in KernelClient.wait_for_ready
         """
-        time.sleep(1)
         return super(SignalTestKernel, self).kernel_info_request(*args, **kwargs)
 
 class SignalTestApp(IPKernelApp):
     kernel_class = SignalTestKernel
     def init_io(self):
         pass # disable stdout/stderr capture
-    
+
 if __name__ == '__main__':
+    time.sleep(2)
```

Comment:
Just clarifying that this is just adding a delay so the object constructor finishes before the test is launched.